defaults/main.yml

```diff
@@ -1,8 +1,10 @@
 openwisp2_python: python3
 ansible_python_interpreter: /usr/bin/python3
 openwisp2_network_topology: false
+openwisp2_radius: false
 openwisp2_controller_version: "0.7.post1"
 openwisp2_network_topology_version: "0.4"
+openwisp2_radius_version: "0.1"
```

Comment:
I will put effort in speeding up the current work on openwisp-radius and release a 0.2 version so we can start with that one